### PR TITLE
[REF] odoo-shippable: Revert the follow pull request #278

### PR DESCRIPTION
**Pull request by JesusZapata**
This change revert the follow pull request https://github.com/Vauxoo/docker-odoo-image/pull/278 to fix the follow issue https://github.com/Vauxoo/docker-odoo-image/issues/282

With this change now the branch is shown on the prompt 

image

Fix https://github.com/Vauxoo/docker-odoo-image/issues/282

### CODE REVIEW

**Comment by JesusZapata** [comment]
@hugho-ad @zaoral @moylop260 

Thanks for report this mistake, in this pull request https://github.com/Vauxoo/docker-odoo-image/pull/283 we found the fix 

If you want test this fix you can open your `.bashrc` and the first lines cut a put in the end of this file

image

image

And logout of the container and login again

File: odoo-shippable/scripts/build-image.sh [hunk 1]
```diff
@@ -419,16 +419,13 @@ usermod -s /bin/bash root
...
     echo "Export the PYTHONPATH IN ${BASHRC}"
-    sed -i '1 i\# Python path \
-if [ "x${TRAVIS_PYTHON_VERSION}" == "x"  ] ; then \
-    TRAVIS_PYTHON_VERSION="2.7" \
-fi \
-source /.repo_requirements/virtualenv/python${TRAVIS_PYTHON_VERSION}/bin/activate \
-source /.repo_requirements/virtualenv/nodejs/bin/activate \
-PYTHONPATH=${PYTHONPATH}:${REPO_REQUIREMENTS}/odoo\n\n' $BASHRC
 	cat >> $BASHRC << EOF
+if [ "x\${TRAVIS_PYTHON_VERSION}" == "x"  ] ; then
```
Comment (moylop260) [inline odoo-shippable/scripts/build-image.sh]: you need scape $?

Comment (moylop260) [reply odoo-shippable/scripts/build-image.sh]: hello?

Comment (JesusZapata) [reply odoo-shippable/scripts/build-image.sh]: @moylop260 
I just copy and paste the follow commit https://github.com/Vauxoo/docker-odoo-image/blob/611fdba993b8764f07cbf16323a71974c53f5ff5/odoo-shippable/scripts/build-image.sh

This commit is previous to my PR https://github.com/Vauxoo/docker-odoo-image/pull/278

**Comment by hbto** [comment]
What is the ETA to land on stable for this PR
Regards

**Comment by JesusZapata** [comment]
@moylop260 @hbto 
I just revert the follow pull request https://github.com/Vauxoo/docker-odoo-image/pull/278

**Pull request merged**